.local.dic

```diff
@@ -297,56 +199,92 @@ geocode/GS
 geocoding
 geolocation
 getter/nombrable
+Getters
+GIFs
+GitHub
+GitLab
 Glimmer
 globals
+Globals
 globbing
+GraphQL
 guidemaker
 hackable
+HammerJS
+Handlebars
 handler
 hardcode
 hardcode/D
-hasMany
 hashchange
+hasMany
 help
 helper/nombrable
+Heroku
+Hm
+Hoc
+Homebrew
 hook/nombrable
 hostname
+HTML5
 htmlbars
+HTMLBars
 htmlbars-inline-precompile
 http
-iOS
+HTTPS
+IE
+IE11
 iframe/nombrable
 in-depth
 in-repo
+IndexedDB
+Inflector
 initialiseur/nombrable
 initializer/nombrable
+Initializer/nombrable
```

Comment:
For instance, not sure the one with an uppercase `I` is still useful with `/nombrable`, we couldn't spot this specific kind of duplication with the initial sorting, so I prefer your new one.